### PR TITLE
Update V4DBHandler

### PR DESCRIPTION
**Pull request by yadach**
## What?
- Add the following methods. (Use the metadata `_kind`.)
  - handler.add_file
  - handler.remove_file
  - handler.add_record
  - handler.remove_record
- Update CLI commands associated with the above.

## Why?
- To manage `{_kind: file}` and `{_kind: record}` explicitly separately.

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/100#" title="Author self-approved">yadach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [yadach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by d-hayashi — COMMENTED** [review]
実装あざます！
LGTMですが、もし余裕があったら `_add_record` と `_add_file` の関数のテストを `test_db.py` に追加していただけるとありがたいです！

File: pydtk/db/v4/handlers/__init__.py
Comment (d-hayashi) [inline pydtk/db/v4/handlers/__init__.py:631]: うまく伝えられていなくてすいません、`record`, `file` の概念はメタデータ用のハンドラ (V4MetaDBHandler) にのみ存在するので、ここの L631-708 をそのまま MetaDBHandler の方に移動できます？

**Review by d-hayashi — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/pydtk/pull/100#pullrequestreview-816446985" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/pydtk/pull/100#" title="Author self-approved">yadach</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fpydtk).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/pydtk/blob/master/OWNERS)~~ [d-hayashi,yadach]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
